recipes/corona_lineage_dynamics/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+${CXX} --std=c++17 scripts/count_lineages_country.cpp -o scripts/count_lineages_country -lstdc++ -lboost_regex ${CXXFLAGS} ${LDFLAGS}
+
+Rscript -e 'install.packages(c("lifecycle", "scales", "withr", "knitr"), repos="https://cloud.r-project.org");'
```

Comment:
Why not use Conda dependencies ?
- https://anaconda.org/conda-forge/r-knitr
- https://anaconda.org/conda-forge/r-withr
- https://anaconda.org/conda-forge/r-scales
- https://anaconda.org/conda-forge/r-lifecycle

Reply:
Thanks. We moved all these into the dependencies.